dynamiqs/time_array.py

```diff
@@ -222,6 +222,21 @@ def reshape(self, *new_shape: int) -> TimeArray:
             New time-array object with the given shape.
         """
 
+    def squeeze(self, axis: int) -> TimeArray:
+        """Squeeze a time-array.
+
+        Args:
+            axis: Axis to squeeze.
+
+        Returns:
+        New time-array object with squeezed_shape
```

Comment:
```suggestion
        Returns:
            New time-array object with squeezed shape.
```
If possible it'd be great to render the doc when you modify it, to check that it looks good 😊

Reply:
Right, sorry I will